en/installation-guide/requirements.rst

```diff
@@ -3,7 +3,8 @@
 Requirements
 =======================
 
-The following Python package (installed using `pip <https://pypi.org/project/pip/>`_) are required:
+The following Python packages (installed using `pip <https://pypi.org/project/pip/>`_) are required:
 
 * `PUM <https://github.com/opengisch/pum>`_ for model upgrades
-* `Pirogue <https://github.com/opengisch/pirogue>`_ for model generation and upgrades
\ No newline at end of file
+* `Pirogue <https://github.com/opengisch/pirogue>`_ for model generation and upgrades
+Note: not needed for the installation, see https://github.com/QGEP/docs/issues/107#issuecomment-572481141
```

Comment:
I think we want people to install these (for a long term stable running of QGEP). I think that this link will be more confusing than helpful in the future once the installation is clearly documented, no?

Reply:
ok for me - so leave it out
Your comment just helped me not to get stuck before even starting the installation.

Reply:
Maybe let's put it like this?

```
.. note::

  These modules are not required for the initial installation but only for subsequent datamodel upgrades.
```